src/webots/sound/WbSoundEngine.cpp

```diff
@@ -93,7 +94,7 @@ static void init() {
   WbLog::toggle(stderr);  // we want to disable stderr to avoid warnings in the console
   try {
     const ALCchar *defaultDeviceName = alcGetString(NULL, ALC_DEFAULT_DEVICE_SPECIFIER);
-    if (defaultDeviceName == NULL)
+    if (defaultDeviceName == NULL || !strcmp(defaultDeviceName, "Null Audio Device"))
```

Comment:
We usually prefer this variant which is arguably more understandable:
```suggestion
    if (defaultDeviceName == NULL || strcmp(defaultDeviceName, "Null Audio Device") == 0)
```